yarn-project/archiver/src/archiver/memory_archiver_store/l1_to_l2_message_store.ts

```diff
@@ -64,6 +64,6 @@ export class L1ToL2MessageStore {
         return indexInTheWholeTree;
       }
     }
-    throw new Error(`L1 to L2 message index not found in the store for message ${l1ToL2Message.toString()}`);
+    return undefined;
```

Comment:
Returning undefined now because that is how other endpoints returning indices behave and it makes it more convenient for the implementation of the new `isL1ToL2MessageSynced` func on aztec node.